opercap.f90

```diff
@@ -0,0 +1,422 @@
+                        RFuncConst = RP1(mnuc,tau,taup,term_R-1,j_chi,coupling_Array)
+                    case (7)
+                        RFuncConst = RD(mnuc,tau,taup,term_R-1,j_chi,coupling_Array)
+                    case (8)
+                        RFuncConst = RS1D(tau,taup,term_R-1,j_chi,coupling_Array)
+                    case default
+                        RFuncConst = 0.
+                        print*, "Um, I ran out of R functions to choose from?"
+                    end select
+
+                    !!!!!!!!!!!!!!!!!!!!!! if I check here to see if RFuncConst is zero, I could skip the integrator call, no?
+                    !!!!!!!!!!!!!!!!!!!!!! what ever pops out of the integrator is just multiplied by this term, so if it's zero it'll kill the integrator's result off (and move on to the next iteration of the loop)
+                    if (RFuncConst.ne.0.) then
+                        qOffset = 0 ! qOffset is used to tell the GFFI how many extra q^2 terms come from the current R function term
+                        if (funcType.gt.3) then ! note that the functions following SigmaPrime in the P_tot sum are all multiplied by an extra factor of q^2/M_N 
+                            qOffset = qOffset + 1
+                        end if
+                        if ((term_R.eq.3) .or. (term_R.eq.4)) then ! add on the single q^2 term
+                            qOffset = qOffset + 1
+                        else if ((term_R.eq.5) .or. (term_R.eq.6)) then ! add on the two q^2 terms
+                            qOffset = qOffset + 2
+                        end if
+                        
+                        w_shared = .false.
+                        if((term_R.eq.2).or.(term_R.eq.4).or.(term_R.eq.6)) then ! if the current R term is one with a v_T^2 in it, this accounts for the v^2/c^2 upon expansion (we deal with the negative just after this do loop)
+                            w_shared = .true.
+                        end if
+
+                        !Loop over the different elements
+                        do eli = 1, niso
+                            a = AtomicNumber_oper(eli)
+                            a_shared = a !make accessible via the module
+
+                            mu = mdm/(mnuc*a)
+                            muplus = (1.+mu)/2.
+                            muminus = (mu-1.d0)/2.
+
+                            J = AtomicSpin_oper(eli)
+                            !!!!!!!!!!!!!!!!!!!!!!!!!simplify yConverse mathematical opperation to help loop speed?
+                            ! yConverse = 2/3.*((0.91*(mnuc*a)**(1./3)+0.3)*10**-13)**2/(2*hbar*c0)**2 !the conversion factor between q and y: y = yConverse * q^2
+                            yConverse = (1.38E-27) * ( ((mnuc*a)**(1./3) + 0.33) / (hbar*c0) )**2 !should be less computationally demanding I think, unless the 10**-27 makes floats do weird stuff - could combine the 10**-27 with the (hbar*c0)**2 to get a less extreme float prefactor
```

Comment:
To be consistent with Catena should be
yConverse = 2.671223d2/(45.d0*a**(-1./3.) - 25.d0*a**(-2./3.))

Reply:
We found it should be:
`yConverse = 264.114/(45.d0*A**(-1./3.)-25.d0*A**(-2./3.))`